### PR TITLE
Alerts in correlations feature documentation

### PR DESCRIPTION
**Pull request by riysaxen-amzn**
### Description
* Adds documentation for two new Correlation Alerts APIs
* Changes the description in Alerts Page to include Correlation Alerts List

### Issues Resolved
[_List any issues this PR will resolve, e.g. Closes [...]._](https://github.com/opensearch-project/security-analytics/issues/988)

### Version
2.15

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by kolchfa-aws** [comment]
@vagimeli Do you mind reviewing this PR?

**Review by vagimeli — APPROVED** [review]
@riysaxen-amzn Doc review complete. Rewrote text for clarity, conciseness, and style guide compliance. Moving to editorial. I'll address editorial comments when that review is completed. Thank you.

File: _security-analytics/api-tools/correlation-eng.md [hunk 1]
```diff
@@ -222,3 +229,136 @@ GET /_plugins/_security_analytics/findings/correlate?finding=425dce0b-f5ee-4889-
...
+        }
+    ],
+    "total_alerts": 2
+}
+```
+{% include copy-curl.html %}
+
+## Acknowledge correlation alerts
+
+You can use the following API to acknowledge the correlation alerts:
```
Comment (natebower) [inline _security-analytics/api-tools/correlation-eng.md:305]:
```suggestion
You can use the following API to acknowledge the correlation alerts.
```

File: _security-analytics/usage/alerts.md [hunk 2]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
 
 # Working with alerts
 
-The Alerts window includes features for viewing and working with alerts. The two main features are:
-* The bar graph with alert information arranged by count, date, and alert status or alert severity.
-* The Alerts list arranged by time of the alert, the alert's trigger name, which detector triggered it, and other details.
+The **Alerts** window provides features to view and manage alerts. It consists of:
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:10]:
```suggestion
The **Alerts** window provides features for viewing and managing alerts. It consists of:
```

File: _security-analytics/usage/alerts.md [hunk 3]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
 
-You can select the **Refresh** button at any time to refresh information on the Alerts page.
+- A bar graph displaying alert counts, dates, and statuses or severities.
+- Two tabs:
+  - **Findings**: Lists detector-generated alerts, showing details such as alert time, trigger name, and detector that triggered the alert.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:14]:
```suggestion
  - **Findings**: Lists detector-generated alerts, showing details such as alert time, trigger name, and the detector that triggered the alert.
```

File: _security-analytics/usage/alerts.md [hunk 4]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
+  - **Correlations**: Lists correlation rule-generated alerts, showing details such as alert time, trigger name, and correlation rule that triggered the alert.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:15]:
```suggestion
  - **Correlations**: Lists correlation-rule-generated alerts, showing details such as alert time, trigger name, and the correlation rule that triggered the alert.
```

File: _security-analytics/usage/alerts.md [hunk 5]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
+
+You can select the **Refresh** button to refresh information on the **Alerts** page.
 
 ---
-## The Alerts graph
 
-The Alerts graph can display alerts by their status or severity. Use the **Group by** dropdown list to specify either Alert status or Alert severity.
+## Alerts graph
+
+The **Alerts** graph displays alerts by their status or severity. Use the **Group by** dropdown list to specify either **Alert status** or **Alert severity**.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:23]:
```suggestion
The **Alerts** graph displays alerts by their status or severity. Use the **Group by** dropdown menu to specify either **Alert status** or **Alert severity**.
```

File: _security-analytics/usage/alerts.md [hunk 6]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
 
-To specify the date range you would like the graph to display, first select the calendar dropdown arrow. The date selector window opens.
+To specify a date range, select the calendar icon to activate the dropdown menu. The date selector window then opens. An example window is shown in the following image
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:25]:
```suggestion
To specify a date range, select the calendar icon to activate the dropdown menu. The date selector window opens. An example window is shown in the following image.
```

File: _security-analytics/usage/alerts.md [hunk 7]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
 <br><img src="{{site.url}}{{site.baseurl}}/images/Security/find-date-pick.png" alt="Date selector for findings graph" width="55%">
 
-You can use the **Quick select** settings to specify an exact window of time.
-* Select either **Last** or **Next** in the first dropdown list to set the window of time behind the current setting or ahead of the current setting.
-* Select a number in the second dropdown list to define a value for the range.
-* Select a unit of time in the third dropdown list. Available options are seconds, minutes, hours, days, weeks, months, and years.
-Select the **Apply** button to apply the range of dates to the graph. Information on the graph changes accordingly.
+You can use the **Quick select** settings to specify a date range.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:28]: Should this end in a colon rather than a period?

File: _security-analytics/usage/alerts.md [hunk 8]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
+* Select either **Last** or **Next** from the first dropdown list to set the date range behind the current setting or ahead of the current setting.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:29]:
```suggestion
* Select either **Last** or **Next** from the first dropdown menu to set the date range to either behind the current setting or ahead of the current setting.
```

File: _security-analytics/usage/alerts.md [hunk 9]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
+* Select a number from the second dropdown list to define a value for the range.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:30]:
```suggestion
* Select a number from the second dropdown menu to define a value for the range.
```

File: _security-analytics/usage/alerts.md [hunk 10]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
+* Select a unit of time from the third dropdown list. Available options are seconds, minutes, hours, days, weeks, months, and years.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:31]:
```suggestion
* Select a unit of time from the third dropdown menu. Available options are seconds, minutes, hours, days, weeks, months, and years.
```

File: _security-analytics/usage/alerts.md [hunk 11]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
+* Select the **Apply** button to apply the time range to the graph. Information on the graph updates immediately
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:32]:
```suggestion
* Select the **Apply** button to apply the time range to the graph. The graph is immediately updated.
```

File: _security-analytics/usage/alerts.md [hunk 12]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
+You can use the left and right arrows in the upper-left corner to shift the time range backward or forward, respectively. When you use these arrows, the start date and end date appear in the date range field. You can then select each one to set an absolute, relative, or current date and time. For absolute and relative changes, select the **Update** button to apply the changes. 
+
+An example window is shown in the following image.
 
 <img src="{{site.url}}{{site.baseurl}}/images/Security/date-pick.png" alt="Altering date range" width="55%">
 
-As an alternative, you can select an option in the **Commonly used** section (see the preceding image of the calendar dropdown list) to conveniently set a window of time. Options include date ranges such as **Today**, **Yesterday**, **this week**, and **week to date**. 
+As an alternative, you can select an option in the **Commonly used** section (see the preceding image of the calendar dropdown list) to conveniently set a date range. Options include **Today**, **Yesterday**, **this week**, and **week to date**.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:44]:
```suggestion
As an alternative, you can select an option in the **Commonly used** section (see the preceding image of the calendar dropdown menu) to conveniently set a date range. Options include **Today**, **Yesterday**, **this week**, and **week to date**. 
```

File: _security-analytics/usage/alerts.md [hunk 13]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
+As an alternative, you can select an option in the **Commonly used** section (see the preceding image of the calendar dropdown list) to conveniently set a date range. Options include **Today**, **Yesterday**, **this week**, and **week to date**. 
 
-When one of the commonly used windows of time is selected, you can select the **Show dates** label in the date range field to populate the range of dates. Following that, you can select either the start date or end date to specify by an absolute, relative, or current date and time setting. For absolute and relative changes, select the **Update** button to apply the changes.
+When a commonly used date range is selected, you can select the **Show dates** label in the date range field to populate the ranges. You can then select either the start date or end date to specify by an absolute, relative, or current date and time setting. For absolute and relative changes, select the **Update** button to apply the changes.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:46]:
```suggestion
When a commonly used date range is selected, you can select the **Show dates** label in the date range field to populate the ranges. You can then select either the start date or end date to specify an absolute, relative, or current date and time setting. For absolute and relative changes, select the **Update** button to apply the changes.
```

File: _security-analytics/usage/alerts.md [hunk 14]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
 
-As one more alternative, you can select an option from the **Recently used date ranges** section to go back to a previous setting.
+You can also select an option from the **Recently used date ranges** section to go back to a previous setting.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:48]: "go back to" => "revert to"?

File: _security-analytics/usage/alerts.md [hunk 15]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
 ---
-## The Alerts list
 
-The Alerts list displays all findings according to the time when the alert was triggered, the alert's trigger name, the detector that triggered the alert, the alert status, and alert severity.
-Use the **Alert severity** dropdown list to filter the list of alerts by severity. Use the **Status** dropdown list to filter the list by alert status.
+## Alerts list
+
+The **Alerts list** displays all alerts, with two tabs for different types of alerts:
 
+- **Findings**: The **Alerts list** displays all findings according to the time when the alert was triggered, the alert's trigger name, the detector that triggered the alert, the alert status, and alert severity.
```
Comment (natebower) [inline _security-analytics/usage/alerts.md:56]:
```suggestion
- **Findings**: The **Alerts list** displays all findings according to the time when the alert was triggered, the alert's trigger name, the detector that triggered the alert, the alert status, and the alert severity.
```

**Review by natebower — COMMENTED** [review]
@vagimeli @riysaxen-amzn Please see my comments and changes and let me know if you have any questions. Thanks!

File: _security-analytics/usage/alerts.md [hunk 16]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
 
-When one of the commonly used windows of time is selected, you can select the **Show dates** label in the date range field to populate the range of dates. Following that, you can select either the start date or end date to specify by an absolute, relative, or current date and time setting. For absolute and relative changes, select the **Update** button to apply the changes.
+When a commonly used date range is selected, you can select the **Show dates** label in the date range field to populate the ranges. You can then select either the start date or end date to specify an absolute, relative, or current date and time setting. For absolute and relative changes, select the **Update** button to apply the changes.
 
-As one more alternative, you can select an option from the **Recently used date ranges** section to go back to a previous setting.
+You can also select an option from the **Recently used date ranges** section to go back to a previous setting.
```
Comment (vagimeli) [inline _security-analytics/usage/alerts.md:48]:
```suggestion
You can also select an option from the **Recently used date ranges** section to revert to a previous setting.
```

File: _security-analytics/usage/alerts.md [hunk 17]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
-To specify the date range you would like the graph to display, first select the calendar dropdown arrow. The date selector window opens.
+To specify a date range, select the calendar icon to activate the dropdown menu. The date selector window opens. An example window is shown in the following image.
 <br><img src="{{site.url}}{{site.baseurl}}/images/Security/find-date-pick.png" alt="Date selector for findings graph" width="55%">
 
-You can use the **Quick select** settings to specify an exact window of time.
-* Select either **Last** or **Next** in the first dropdown list to set the window of time behind the current setting or ahead of the current setting.
-* Select a number in the second dropdown list to define a value for the range.
-* Select a unit of time in the third dropdown list. Available options are seconds, minutes, hours, days, weeks, months, and years.
-Select the **Apply** button to apply the range of dates to the graph. Information on the graph changes accordingly.
+You can use the **Quick select** settings to specify a date range.
```
Comment (vagimeli) [inline _security-analytics/usage/alerts.md:28]:
```suggestion
You can use the **Quick select** settings to specify a date range:
```

File: _security-analytics/usage/alerts.md [hunk 18]
```diff
@@ -7,41 +7,53 @@ nav_order: 46
...
-## The Alerts list
 
-The Alerts list displays all findings according to the time when the alert was triggered, the alert's trigger name, the detector that triggered the alert, the alert status, and alert severity.
-Use the **Alert severity** dropdown list to filter the list of alerts by severity. Use the **Status** dropdown list to filter the list by alert status.
+## Alerts list
+
+The **Alerts list** displays all alerts, with two tabs for different types of alerts:
+
+- **Findings**: The **Alerts list** displays all findings according to the time when the alert was triggered, the alert's trigger name, the detector that triggered the alert, the alert status, and the alert severity.
+- **Correlations**:  The **Alerts list** displays all correlations according to the rule and rule time window when the alert was triggered, the alert's trigger name, the correlation rule name that triggered the alert, the alert status, and the alert severity.
```
Comment (vagimeli) [inline _security-analytics/usage/alerts.md:57]:
```suggestion
- **Correlations**:  The **Alerts list** displays all correlations, including the correlation rule and time window, the alert's trigger name, the correlation rule name that triggered the alert, the alert status, and the alert severity.
```

**Comment by vagimeli** [comment]
> @vagimeli @riysaxen-amzn Please see my comments and changes and let me know if you have any questions. Thanks!

@natebower @riysaxen-amzn I've accepted editorial changes and revised text to address comments.

**Pull request merged**